Resolve #1

mstackhouse opened the pull request:
Blah 

Closes #1

mstackhouse committed:
Resolve #1

Fmenius commented on R/app_ui.R:
```diff
@@ -10,7 +10,8 @@ app_ui <- function(request) {
     # Your application UI logic
     fluidPage(
-      golem::golem_welcome_page() # Remove this line to start building your UI
+      h1("Hello!"),
```
Test comment make this prettier
```suggestion
      h1("Hello!  YAY!"),
```

mstackhouse committed:
Update R/app_ui.R

Co-authored-by: Frank Menius <frank.menius@atorusresearch.com>

mstackhouse committed:
Added useful text.

mstackhouse committed:
Whoops.

Merge branch 'bugfix/issue-1' of github.com:atorus-research/shinyapp into bugfix/issue-1

# Conflicts:
#	R/app_ui.R
